src/frontend/src/lib/types/token.ts

```diff
@@ -70,11 +70,11 @@ export type TokenUi = Token & TokenFinancialData;
 export type OptionTokenUi = Option<TokenUi>;
 
 // TODO: remove header and nativeNetwork, since we added nativeToken that has all their data, and they became redundant
-export interface TokenGroupUi {
+export type TokenGroupUi = {
```

Comment:
unrelated: why is this type suffixed with `Ui` btw?

Reply:
it is to indicate that the group contains TokenUi and not just Token: in fact it should be used only in UI context (appearance, show the list, etc)

Reply:
so, shouldn't it be called `TokenUiGroup`?

Reply:
right! make sense! can i do it in another PR? because I am pretty sure it will change some other components and should be indeed a different topic

Reply:
definitely another PR, thanks

Reply:
Done:

- PR #3075